src/main/java/frc/robot/poseestimator/limelights/SmartLimelightsConfig.java

```diff
@@ -0,0 +1,10 @@
+package frc.robot.poseestimator.limelights;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+
+public record SmartLimelightsConfig(
+	String Logpath,
+	String HardwareLogpath,
```

Comment:
```suggestion
	String hardwareLogpath,
```
create constrctor that create it using the logpath